src/main/java/uk/gov/pay/api/resources/SearchRefundsResource.java

```diff
@@ -66,17 +66,27 @@ public SearchRefundsResource(SearchRefundsService searchRefundsService, PublicAp
     )
     public SearchRefundsResults searchRefunds(@Parameter(hidden = true)
                                   @Auth Account account,
-                                  @Parameter(description = "From date of refunds to be searched (this date is inclusive). Example=2015-08-13T12:35:00Z")
+                                  @Parameter(description = "Returns refunds created on or after the `from_date`. " +
+                                          "Date and time must use Coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - `YYYY-MM-DDTHH:MM:SSZ`.")
                                   @QueryParam("from_date") String fromDate,
-                                  @Parameter(description = "To date of refunds to be searched (this date is exclusive). Example=2015-08-14T12:35:00Z")
+                                  @Parameter(description = "Returns refunds created before the `to_date`. " +
```

Comment:
Same as above for date format

Reply:
thanks @kbottla I will feed that back to Nathan and get the changes made

Reply:
date format updated